tests/single-node/test.sh

```diff
@@ -0,0 +1,105 @@
+top_srcdir="$(realpath "$(pwd)/../..")"
+
+ANSIBLE_FORCE_COLOR=true
+export ANSIBLE_FORCE_COLOR
+
+make_shell() {
+        make --no-print-directory -C "${top_srcdir}" shell C="$*"
+}
+
+die() {
+        exit 1
+}
+
+setup_suite() {
+        echo "Creating loopback block-device and configuring VM"
+        sudo truncate -s 20G /kube-lvm || die
+        sudo losetup /dev/loop0 /kube-lvm || die
+
+        echo "Disabling iptables"
+        sudo systemctl disable --now iptables || die
+        sudo systemctl disable --now ip6tables || die
+        sudo iptables -F || die
+        sudo iptables -X || die
+
+        echo "Creating shell environment"
+        make_shell true
+
+        echo "Deploy the cluster"
+        make_shell ansible-playbook -i "$(pwd)/inventory" metal-k8s.yml --skip elasticsearch || die
+
+        KUBECONFIG=$(pwd)/inventory/artifacts/admin.conf
+        export KUBECONFIG
+
+        echo "Run a sample kubectl command"
+        make_shell kubectl get nodes
+}
+
+test_reclaim_storage() {
+        echo "Listing all PVs before test (some should be available)"
+        make_shell kubectl get pv
+
+        echo "Running simple Pod with PersistentVolumeClaim"
+        make_shell kubectl apply -f - << EOF
+kind: PersistentVolumeClaim
```

Comment:
Those file has meaning outside of the test, could be nice to have it as separated resources. This break a little the "self-contain" test, but allow us to launch the pod on any cluster without reclaiming the entire storage of the cluster

Reply:
I'm not sure why you'd want to run this specific Pod & PVC on 'any cluster'? In that case, copy-pasting into some file as a one-off seems reasonable :)